next.config.js

```diff
@@ -125,7 +125,28 @@ module.exports = millionLint({
       ...interests.map((interest) => {
         return {
           source: `/${interest}/:tool(dashboard|reports|contributors|activity)`,
-          destination: `/explore/topic/${interest}/:tool`,
+          destination: `/explore`,
+          permanent: true,
+        };
+      }),
+      ...interests.map((interest) => {
+        return {
+          source: `/explore/topic/${interest}/:tool(dashboard|reports|contributors|activity)`,
+          destination: `/explore`,
+          permanent: true,
+        };
+      }),
+      ...interests.map((interest) => {
+        return {
+          source: `/explore/topic/${interest}`,
+          destination: `/explore`,
+          permanent: true,
+        };
+      }),
```

Comment:
Could this just be

```suggestion
      {
        source: `/explore/topic/*`,
        destination: `/explore`,
        permanent: true,
      },
```

Reply:
Yep, I was just mimicking existing behavior